oteapi_dlite/strategies/parse.py

```diff
@@ -101,6 +111,25 @@ def get(
         config = self.parse_config.configuration
         cacheconfig = config.datacache_config
 
+        for addition in config.global_configuration_additions.storage_path:
+            dlite.storage_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.storage_plugin_path:
+            dlite.storage_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.mapping_plugin_path:
+            dlite.mapping_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.python_storage_plugin_path:
+            dlite.python_storage_plugin_path.append(addition)
+        for (
+            addition
+        ) in config.global_configuration_additions.python_mapping_plugin_path:
+            dlite.python_mapping_plugin_path.append(addition)
```

Comment:
Same as above, can be fractured out into an utility function.

Reply:
Will keep the discussion to the previous comment then, resolving this conversation.